<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1726</width>
    <height>1152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="locationBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>631</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Location</string>
    </property>
    <widget class="QComboBox" name="stateBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QListWidget" name="cityBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>211</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="zipcodeBox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>221</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>City</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Zipcode</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="zipstatBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>1071</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Zipcode Statistics</string>
    </property>
    <widget class="QLabel" name="topCategoryLabel">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Top Categories</string>
     </property>
    </widget>
    <widget class="QListWidget" name="numBusiness">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="totalPop">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="avgIncome">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string># of Businesses</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Population</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Average Income</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="topCategoryTable">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>691</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="filterCategoryBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>1711</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter on Categories</string>
    </property>
    <widget class="QListWidget" name="categoryList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>331</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="categoryLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Category</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="businessTable">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>1311</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="businessLabel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>35</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Businesses</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="popularBusinessTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>871</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="successfulBusinessTable">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>690</y>
      <width>801</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="popBusinessLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>649</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Popular Businesses (in zipcode)</string>
    </property>
   </widget>
   <widget class="QLabel" name="succBusinessLabel">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>640</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Successful Businesses (in zipcode)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1726</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBusiness_Page">
    <property name="title">
     <string>Business Page</string>
    </property>
   </widget>
   <addaction name="menuBusiness_Page"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>stateBox</tabstop>
  <tabstop>cityBox</tabstop>
  <tabstop>zipcodeBox</tabstop>
  <tabstop>categoryList</tabstop>
  <tabstop>businessTable</tabstop>
  <tabstop>popularBusinessTable</tabstop>
  <tabstop>successfulBusinessTable</tabstop>
  <tabstop>numBusiness</tabstop>
  <tabstop>totalPop</tabstop>
  <tabstop>avgIncome</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
